lib/livebook_web/live/apps_live.ex

```diff
@@ -41,170 +41,184 @@ defmodule LivebookWeb.AppsLive do
+                        ]}
+                        aria-label="open app"
+                        href={~p"/apps/#{app.slug}/#{app_session.id}"}
+                      >
+                        <.remix_icon icon="link" class="text-lg" />
+                      </a>
+                    </span>
+                    <span class="tooltip left" data-tooltip="Debug">
+                      <a
+                        class="icon-button"
+                        aria-label="debug app"
+                        href={~p"/sessions/#{app_session.id}"}
+                      >
+                        <.remix_icon icon="terminal-line" class="text-lg" />
+                      </a>
+                    </span>
+                    <%= if app_session.app_status.lifecycle == :active do %>
+                      <span class="tooltip left" data-tooltip="Deactivate">
+                        <button
+                          class="icon-button"
+                          aria-label="deactivate app session"
+                          phx-click={
+                            JS.push("deactivate_app_session",
+                              value: %{slug: app.slug, session_id: app_session.id}
+                            )
+                          }
+                        >
+                          <.remix_icon icon="stop-circle-line" class="text-lg text-brand-pink" />
+                        </button>
+                      </span>
+                    <% else %>
+                      <span class="tooltip left" data-tooltip="Terminate">
+                        <button
+                          class="icon-button"
+                          aria-label="terminate app session"
+                          phx-click={
+                            JS.push("terminate_app_session",
+                              value: %{slug: app.slug, session_id: app_session.id}
+                            )
+                          }
+                        >
+                          <.remix_icon icon="delete-bin-6-line" class="text-lg" />
+                        </button>
+                      </span>
+                    <% end %>
+                  </:actions>
+                </.grid>
               <% else %>
-                <span class="tooltip left" data-tooltip="Terminate">
-                  <button
-                    class="icon-button"
-                    aria-label="terminate app session"
-                    phx-click={
-                      JS.push("terminate_app_session",
-                        value: %{slug: app.slug, session_id: app_session.id}
-                      )
-                    }
-                  >
-                    <.remix_icon icon="delete-bin-6-line" class="text-lg" />
-                  </button>
-                </span>
+                <div class="p-4 uppercase text-gray-500 text-sm font-medium leading-normal tracking-wider">
+                  NO RUNNING SESSIONS
```

Comment:
```suggestion
                  No running sessions
```

(since we uppercase via css anyway :)